app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt

```diff
@@ -18,6 +18,7 @@ import chat.rocket.core.model.ChatRoom
 import com.facebook.drawee.view.SimpleDraweeView
 import kotlinx.android.synthetic.main.avatar.view.*
 import kotlinx.android.synthetic.main.item_chat.view.*
+import kotlinx.android.synthetic.main.unread_messages_badge.view.*
```

Comment:
We need because `text_total_unread_messages` comes from the `unread_messages_badge` layout.

Note: we'll use the badge on "chat list" view and "chat room" view.